resources/web/style/rtpcontainer.pcss

```diff
@@ -5,19 +5,50 @@
  * on the rtpcontainer in general. */
 #rtpcontainer {
   .mktg-promo {
-    border-radius: 5px;
-    margin-bottom: 40px;
-    padding: 20px;
-    border: 1px solid #ccc;
+    /* border-radius: 5px; */
```

Comment:
Do you want to remove this or keep it? I figure it isn't really a great comment.

Reply:
Thanks for the catch. Remove please :)